hnclient-v2/pom.xml

```diff
@@ -182,6 +182,30 @@
         </configuration>
       </plugin>
 
+	  <plugin>
+	    <groupId>org.apache.maven.plugins</groupId>
+	    <artifactId>maven-assembly-plugin</artifactId>
+	    <executions>
+	        <execution>
+	            <phase>package</phase>
+	            <goals>
+	                <goal>single</goal>
+	            </goals>
+	            <configuration>
+	                <archive>
+	                <manifest>
+	                    <mainClass>
+	                        ca.bc.gov.hlth.hnclientv2.HnsClientMainMethod
+	                    </mainClass>
+	                </manifest>
+	                </archive>
+	                <descriptorRefs>
+	                    <descriptorRef>jar-with-dependencies</descriptorRef>
+	                </descriptorRefs>
+	            </configuration>
+	        </execution>
+	    </executions>
+	</plugin>
```

Comment:
For consistency's sake with HN secure we might want to look into using the shade plugin instead. I believe it offers more fine grained control over the executable jar. But I could go either way. I opened this up to feedback on the Developer channel.